exercises/intergalactic-transmission/canonical-data.json

```diff
@@ -0,0 +1,440 @@
+{
+  "exercise": "intergalactic-transmission",
+  "comments": ["Inputs and expected data are given in hexadecimal"],
+  "cases": [
+    {
+      "description": "calculate transmit sequences",
+      "cases": [
+        {
+          "uuid": "f99d4046-b429-4582-9324-f0bcac7ab51c",
+          "description": "empty message",
+          "property": "transmitSequence",
+          "input": {
+            "message": []
+          },
+          "expected": []
+        },
+        {
+          "uuid": "ee27ea2d-8999-4f23-9275-8f6879545f86",
+          "description": "0x00 is transmitted as 0x0000",
+          "property": "transmitSequence",
+          "input": {
+            "message": ["0x00"]
+          },
+          "expected": ["0x00", "0x00"]
+        },
+        {
+          "uuid": "97f27f98-8020-402d-be85-f21ba54a6df0",
+          "description": "0x02 is transmitted as 0x0100",
+          "property": "transmitSequence",
+          "input": {
+            "message": ["0x02"]
+          },
+          "expected": ["0x01", "0x00"]
```

Comment:
`0x02` is `0000 0010`. Pad that to 7-bit chunks and you get `0000 001_ | 0000 000_`. Add parity bits for `0000 0011 | 0000 0000`. Shouldn't the output be `0x03 0x00`?

Reply:
You're right, it should be `0x03 0x00`! I'll update the data.

Reply:
```
» python intergalactic_transmission.py https://raw.githubusercontent.com/exercism/problem-specifications/eb8ef7696498709799f1ff6830189ab23c471d17/exercises/intergalactic-transmission/canonical-data.json
{'pass': 26, 'fail': 0}
```

My solution says the data is correct :D Or, the data says my solution is correct.